tlg0559.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,3670 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+ <teiHeader xml:lang="eng">
+  <fileDesc>
+   <titleStmt>
+    <title xml:lang="deu">Herons Von Alexandria Druckwerke Und Automatentheater</title>
```

Comment:
This `<title>` element under the `<titleStmt>` should contain the title of the work, not the title of the volume. According to the TLG, the title is "Pneumatica".